<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>301</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simulation settings</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Mexico"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxPreys">
       <property name="title">
        <string>Preys:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelInitialPreys">
            <property name="text">
             <string>Initial percentage of preys:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditInitialPreys">
            <property name="text">
             <string>0.5</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCompetenceFactor">
            <property name="text">
             <string>Competence factor:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditCompetenceFactor">
            <property name="text">
             <string>0.05</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPreyReproductionRadius">
            <property name="text">
             <string>Reproduction radius:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBoxPreyReproductionRadius">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelPreyReproductiveCapacity">
            <property name="text">
             <string>Reproductive capacity:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBoxPreyReproductiveCapacity">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>labelInitialPreys</zorder>
       <zorder>labelPreyReproductionRadius</zorder>
       <zorder>labelCompetenceFactor</zorder>
       <zorder>labelPreyReproductiveCapacity</zorder>
       <zorder>lineEditInitialPreys</zorder>
       <zorder>lineEditCompetenceFactor</zorder>
       <zorder>groupBoxPredators</zorder>
       <zorder>spinBoxPreyReproductionRadius</zorder>
       <zorder>spinBoxPreyReproductiveCapacity</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPredators">
       <property name="title">
        <string>Predators:</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="labelInitialPredators">
            <property name="text">
             <string>Initial number of predators:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEditInitialPredators">
            <property name="text">
             <string>3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="spinBoxMaxSpeed">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="spinBoxPredatorReproductiveCapacity">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="spinBoxPredatorReproductionRadius">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="spinBoxSocialRadius">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEditCognitiveFactor">
            <property name="text">
             <string>0.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEditSocialFactor">
            <property name="text">
             <string>1.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="labelCognitiveFactor">
            <property name="text">
             <string>Cognitive factor:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="labelSocialFactor">
            <property name="text">
             <string>Social factor:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="labelMaxSpeed">
            <property name="text">
             <string>Maximum speed:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="labelPredatorReproductiveCapacity">
            <property name="text">
             <string>Reproductive capacity:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="labelPredatorReproductionRadius">
            <property name="text">
             <string>Reproduction radius:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Social radius:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>labelInitialPredators</zorder>
       <zorder>labelCognitiveFactor</zorder>
       <zorder>labelSocialFactor</zorder>
       <zorder>labelMaxSpeed</zorder>
       <zorder>labelPredatorReproductionRadius</zorder>
       <zorder>labelPredatorReproductiveCapacity</zorder>
       <zorder>label_12</zorder>
       <zorder>lineEditInitialPredators</zorder>
       <zorder>lineEditCognitiveFactor</zorder>
       <zorder>lineEditSocialFactor</zorder>
       <zorder>spinBoxMaxSpeed</zorder>
       <zorder>spinBoxPredatorReproductionRadius</zorder>
       <zorder>spinBoxSocialRadius</zorder>
       <zorder>spinBoxPredatorReproductiveCapacity</zorder>
       <zorder>groupBoxPreys</zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelResults">
       <property name="text">
        <string>Path to results file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
